### PR TITLE
grpc: add -write_secrets flag

### PR DESCRIPTION
**Pull request by tam7t**
grpc: add -write_secrets to control whether to write files or return them in grpc response

Addresses https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/issues/98

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to secrets-store-csi-driver-provider-gcp will be documented
 
 ## unreleased (v0.4.0)
 
+### Added
+
+* `-write_secrets` flag. Set `-write_secrets=false` with `v0.0.21+` of the `secrets-store-csi-driver` to have the driver write the secrets. This will be made the default in `v0.5.0`. [#98](https://github.com/GoogleCloudPlatform/secrets-store-csi-driver-provider-gcp/issues/98)
```
Comment (rlingutla) [inline CHANGELOG.md:9]: `...driver write the secrets.` I thought that 'false' meant that the driver would return the Secrets in the grpc response?

File: main.go [hunk 2]
```diff
@@ -48,6 +48,7 @@ var (
 	metricsAddr   = flag.String("metrics_addr", ":8095", "configure http listener for reporting metrics")
 	enableProfile = flag.Bool("enable-pprof", false, "enable pprof profiling")
 	debugAddr     = flag.String("debug_addr", "localhost:6060", "port for pprof profiling")
+	writeSecrets  = flag.Bool("write_secrets", true, "whether to write the secrets directly (true) or returns secrets in grpc response (false, requires driver v0.20.1)")
```
Comment (rlingutla) [inline main.go:51]: Nit: 'directly' -> 'directly to the filesystem'

**Review by rlingutla — APPROVED** [review]
A few comment nits, otherwise LGTM

File: main.go (hunk 2 above)
Comment (tam7t) [reply main.go:51]: done

File: CHANGELOG.md (hunk 1 above)
Comment (tam7t) [reply CHANGELOG.md]: The "driver" is the `secrets-store-csi-driver` process who made the RPC to our process. So when we return it in the grpc response the driver, not `secrets-store-csi-driver-provider-gcp`, writes the files. Updated to try and clarify

**Review by rlingutla — APPROVED** [review]

**Pull request merged**